src/sonic-config-engine/tests/data/dns/static_dns.json

```diff
@@ -0,0 +1,6 @@
+{
+    "DNS_NAMESERVER": {
```

Comment:
>DNS_NAMESERVER [](http://example.com/codeflow?start=5&length=14)

Need new yang model for this new ConfigDB table.
Also need yang unit test.

Reply:
Yang model and tests are added: https://github.com/sonic-net/sonic-buildimage/pull/13834